eng/Version.Details.xml

```diff
@@ -390,6 +390,10 @@
       <Uri>https://github.com/dotnet/sdk</Uri>
       <Sha>a134a94cfb1209e4181072719d554eb232b51317</Sha>
     </Dependency>
+    <Dependency Name="VS.Redist.Common.NetCore.SdkPlaceholder" Version="9.0.100-preview.5.24272.19">
```

Comment:
These are the available packages. 

![image](https://github.com/dotnet/runtime/assets/19413848/d89c71ab-cff9-4ad3-afa9-87e572d2858b)
```suggestion
    <Dependency Name="VS.Redist.Common.NetCore.SdkPlaceholder.x64.9.0" Version="9.0.100-preview.5.24272.19">
```